backupctl/backupctl.py

```diff
@@ -142,6 +142,7 @@ def main():
     elif args.command == "log":
         history_show(hist)
     else:
+        LOG.error("backupctl: '{}' is not a backupctl command.".format(args.command))
```

Comment:
Can we print the help after this?

Reply:
I would rather add [choices](https://docs.python.org/2/library/argparse.html#choices) to the `add_argument` call for command.